doc/source/_static/tag-mapping.json

```
@@ -0,0 +1,48 @@
+{
+  "scalableBatchInference": "PyTorch,Image Segmentation,Prediction",
+  "batchActorPool": "Prediction",
+  "batchCore": "Prediction",
+  "nycTaxiData": "Prediction",
+  "batchOcr": "Preprocessing",
+  "millionModels": "Regression,Training,Sklearn",
+  "batchTrainingCore": "Regression,Training,Sklearn",
+  "batchTrainingDatasets": "Regression,Training,Sklearn",
+  "tuneBasicParallel": "Regression,Training,Sklearn",
+  "tuneBatch": "Regression,Training,Tuning,Sklearn",
+  "instacartFulfillment": "Training,Prediction",
+  "productionizingMLServe": "Serving",
+  "simplifyMLOpsServe": "Serving",
+  "gettingStartedServe": "Serving",
+  "compositionServe": "Serving",
+  "examplesServe": "Serving",
+  "useCasesServe": "Serving",
+  "gettingStartedTune": "Tuning",
+  "distributeHPOTune": "Tuning",
+  "simpleDistributedHPO": "Tuning",
+  "HPOTransformers": "Tuning,PyTorch,Classification",
+  "examplesTune": "Tuning",
+  "useCasesTune": "Tuning",
+  "pyTorchTrain": "Training,PyTorch",
+  "xgboostTrain": "Training,XGBoost",
+  "gettingStartedTrain": "Training",
+  "trainingTransformers": "Training,PyTorch,Classification,Prediction",
+  "examplesTrain": "Training",
+  "useCasesTrain": "Training",
+  "appliedRLCourse": "Reinforcement Learning",
+  "introRLLib": "Reinforcement Learning",
+  "gettingStartedRLlib": "Reinforcement Learning",
+  "riotRL": "Reinforcement Learning",
+  "examplesRL": "Reinforcement Learning",
+  "useCasesRL": "Reinforcement Learning",
+  "merlin": "Preprocessing,Training,Prediction",
+  "uberScaleDL": "Preprocessing,Training,,PredictionTuning,XGBoost,TensorFlow,PyTorch",
+  "instacartMLPlatformTripled": "Preprocessing,,PredictionTraining,Tuning",
+  "predibase": "Preprocessing,Training,,PredictionTuning,PyTorch",
+  "GKEMLPlatform": "Preprocessing,Training,Prediction,Tuning,TensorFlow,Serving",
+  "summitMLPlatform": "Preprocessing,,PredictionTraining,Tuning,Serving",
+  "torchImageExample": "Preprocessing,,PredictionTraining,PyTorch,Classification",
```

```suggestion
  "uberScaleDL": "Preprocessing,Training,Prediction,Tuning,XGBoost,TensorFlow,PyTorch",
  "instacartMLPlatformTripled": "Preprocessing,Prediction,Training,Tuning",
  "predibase": "Preprocessing,Training,Prediction,Tuning,PyTorch",
  "GKEMLPlatform": "Preprocessing,Training,Prediction,Tuning,TensorFlow,Serving",
  "summitMLPlatform": "Preprocessing,Prediction,Training,Tuning,Serving",
  "torchImageExample": "Preprocessing,Prediction,Training,PyTorch,Classification",
```

seems like there's a bit of a typo here for